website/docs/using-odo/create-component.md

```diff
@@ -3,4 +3,139 @@ title: Create Component
 sidebar_position: 1
 ---
 # Creating components using odo
-This doc explains all the different ways and possibilities for a user to run `odo component create`
\ No newline at end of file
+
+[Component](../getting-started/basics#component) is the most basic unit of operation for odo. And the way to create one is using `odo create` (short for `odo component create`) command.
+
+In simplest terms, when you "create" an odo component, you populate your current working directory with the file `devfile.yaml`. It is a manifest file that contains information about various resources (URL, Storage, Services, etc.) that correspond to your component, and will be created on the Kubernetes cluster when you execute `odo push` command. Most odo commands will first modify (add or remove configuration from) this file, and then subsequent `odo push` will create or delete the resources from the Kubernetes cluster. However, odo users are not expected to know how the `devfile.yaml` is organized; it is the odo commands that would create, update, or delete it.
```

Comment:
>It is a manifest file that

I would rather prefer we say `Devfile is a manifest file` than calling it `It`.

Reply:
>However, odo users are not expected to know how the `devfile.yaml` is organized; it is the odo commands that would create, update, or delete it.

What do you think about highlighting this line in a separate para perhaps or just highlighting it here? It would sort of take away the pressure of learning a new thing i.e. Devfile.

Reply:
Makes sense. Done both.